report/src/main/webapp/app/layout/main/test/mainView.test.view.html

```diff
@@ -17,7 +17,7 @@
     limitations under the License.
 
 -->
-<div class="main page-main">
+<div class="page-main">
```

Comment:
Wasn't `main` class responsible for handling expanding and collapsing sidebar?
Can we safely remove it without losing that functionality? Is `main` class used anywhere now?